lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -0,0 +1,49 @@
+module CC
+  module Engine
+    module Analyzers
+      class EngineConfig
+        attr_reader :config
+
+        def initialize(hash)
+          normalize(hash)
+        end
+
+        def language(language)
+          config.
+            fetch("languages", {}).
```

Comment:
If we're good at providing the default values in `build_language_config`, could we just not do this `fetch` stuff?